### PR TITLE
Media

### PR DESCRIPTION
**Pull request by jyyan**

### CODE REVIEW

File: js/impress-audio.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+Author: Daniel Simons
+Author Email: daniel.simons1@gmail.com
+Version: 1.0.0
+License: Free General Public License (GPL)
+*/
+(function(document, $, undefined) {
+  'use strict';
+
+  var isPlaying, waitTimeout ,
+    audio,video,media,currPlaying,
```
Comment (dawikur) [inline js/impress-audio.js]: Why are you mixing space style? In this line you have no spaces around ',' and in previous line you have.

File: js/impress-audio.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    impressObj = impress(),
+    impressGoto = impressObj.goto;
+  var TimerID;
```
Comment (dawikur) [inline js/impress-audio.js]: Why mixing camelCase and UpperCamelCase of var names?

File: js/impress-audio.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  var SpacingTime = 300; //ms
```
Comment (dawikur) [inline js/impress-audio.js]: Wouldn't 

    var spacigTimeMs = 300;

be better? You don't need to remember it's in ms

**Comment by jyyan** [comment]
its my bad , push into wrong repo , please forget this.
and btw, thanks **Dawid Kurek** help me code review about this , thank you.

**Comment by dawikur** [comment]
Sure :) I started reviewing push and then realized you have already closed it. If you want more comments just ask :smiley:

**Pull request closed without merging**